web-app-router/api/auth/[auth0]/index.ts

```diff
@@ -0,0 +1,38 @@
+import { loginCallback } from "@/api/login-callback";
+
+import {
+  AppRouteHandlerFnContext,
+  HandlerError,
+  handleAuth,
+  handleCallback,
+  handleLogin,
+  handleLogout,
+} from "@auth0/nextjs-auth0";
+
+import { NextRequest, NextResponse } from "next/server";
+
+export const auth0 = handleAuth({
+  login: (req: NextRequest, ctx: AppRouteHandlerFnContext) => {
+    const invite_id = req.nextUrl.searchParams.get("invite_id");
+
+    return handleLogin(req, ctx, {
+      returnTo: invite_id
+        ? `/api/auth/login-callback?invite_id=${invite_id}`
+        : "/api/auth/login-callback",
+    });
+  },
+
+  callback: handleCallback,
+  "login-callback": loginCallback,
```

Comment:
Should leave a TODO here that we need to add delete back

Reply:
Or mark it in linear just so we come back

Reply:
It will be created when I will start onboarding flow pages

Reply:
I've added a comment